src/FlixterHeader.jsx

```diff
@@ -10,15 +11,28 @@ function FlixterHeader({ likedMovies, watchedMovies }) {
   return (
     <header className="App-header">
       <h1 className="title">🎥 Flixter 🎬</h1>
-      <button className="toggleSidebar" onClick={toggleSidebar}>
-        ≡
-      </button>
-      <Sidebar
-        likedMovies={likedMovies}
-        isOpen={isSidebarOpen}
-        toggleSidebar={toggleSidebar}
-        watchedMovies={watchedMovies}
-      />
+      <div>
+        <button className="toggleSidebar" onClick={toggleSidebar}>
+          ≡
+        </button>
+
+        <Sidebar
+          likedMovies={likedMovies}
+          isOpen={isSidebarOpen}
+          toggleSidebar={toggleSidebar}
+          watchedMovies={watchedMovies}
+        />
+        <button
+          className="logOut"
+          onClick={() => {
+            doSignOut().then(() => {
```

Comment:
Are you planning to replace promises with asyn await everywhere? 
Is that something that can be done?

Reply:
I can do that certainly! What is the reason? I'm assuming async await is more clean and easier to trace errors with.

Reply:
IMO It's easier to read, you can prefer promises, though, it's all good. 
The main takeaway is consistency, stick to one of those two options.